libraries/chain/include/eosio/chain/finality/finalizer.hpp

```diff
@@ -148,11 +148,12 @@ namespace eosio::chain {
 
          // then save the safety info and, if successful, gossip the votes
          if (!votes.empty()) {
-            save_finalizer_safety_info();
```

Comment:
do we now ignore any error when saving the file? Even if the `persist_file_path` is empty?

Reply:
We verify `persist_file_path` in `load_finalizer_safety_info` on startup.

Reply:
Still, do we want to ignore any fsi file saving error (just outputting a log message)?

Reply:
That is what @arhag requested.